src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1428,6 +1443,85 @@ export class PatientViewPageStore {
         );
     }
 
+    @computed
+    get annotatedExtendedAlterationData(): AnnotatedExtendedAlteration[] {
```

Comment:
this logic doesn't exist in resultsView?

Reply:
Sort of. This one is tricky. We rely on OQL filter functions (oqlfilter.ts) in the results view. It is not straightforward to completely isolate that logic. It might be possible to create a fake OQL query on the patient view page store but instead I decided to implement a simplified version of this logic for the patient view.